.github/workflows/algolia-publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Use Node
+        uses: actions/setup-node@master
+        with:
+          node-version: 18.x
+          cache: 'yarn'
+
+      - name: Install Dependencies using Yarn
+        run: yarn
+
+      - name: Build Packages
+        run: yarn build
+
+      - name: Typecheck
+        run: yarn workspace @graphql-yoga/website run build
+
+      - name: Algolia push
+        run: yarn workspace @graphql-yoga/website run algolia-sync
```

Comment:
please rename `@graphql-yoga/website` to `website` in all place to make consistency